src/components/owned/OwnerInfo.tsx

```diff
@@ -58,16 +63,18 @@ export default function ({
       </HeaderText>
 
       <BodyText>
-        This is a zkNFT derivative. It means this person has been verified to
-        own at least one ‘
+        {emailRecord
+          ? 'This is a zkNFT derivative of a work email. It means this person has been verified to work at '
+          : 'This is a zkNFT derivative. It means this person has been verified to own at least one '}
+        ‘
         <ExternalLink
-          url={getEtherscanAddressUrl(record.originalContract.address)}
+          url={getEtherscanAddressUrl(record.derivativeContract.address)}
```

Comment:
There should be the original address, if it is not there, then you can display the domain